src/content/docs/authenticate/enterprise-connections/azure.mdx

```diff
@@ -71,17 +69,42 @@ If you are [importing users into Kinde](/manage-users/add-and-edit/import-users
 12. In the **Applications** section, select the applications you want to activate the connection for.
 13. Select **Save**.
 
+## (Optional) Sync groups across to Kinde
+
+### Configure MS Entra app
+
+1. Open your application in the [MS Azure Portal](https://portal.azure.com/).
+2. Go to **Token configuration** in the left menu. 
+3. Select **Add groups claim**.
+4. In the window that appears, select the groups to be included in tokens.
+
+![image of edit groups claim screen](https://imagedelivery.net/skPPZTHzSlcslvHjesZQcQ/fed278be-bdcd-43b6-7130-8c866928b700/public/)
```

Comment:
@clairekinde11 can you remove the backslash for the image URL? Cloudflare will return the image if it contains the backslash.

Reply:
Fixed